pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2207,6 +2207,10 @@ type MultiClusterServiceSpec struct {
 	// like Globalnet Submariner.
 	// +optional
 	Enabled bool `json:"enabled,omitempty"`
+
+	// ClusterID uniquely identifies a cluster. It is used as a prefix to nslookup exported
+	// services. For example: <clusterid>.<svc>.<ns>.svc.clusterset.local
```

Comment:
How does the user know what to set the clusterID to? Would it be found in their kube context? And if the k8s dns lookup can work with it, isn't there a way rook can look up the cluster ID?

Reply:
Ok, so the user must be aware of the clusterID and apply it the same both for submariner and in rook.